sale_automatic_workflow/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,23 @@
+Create workflows with more or less automatization and apply it on sales
+orders.
+
+A workflow can:
+
+- Apply default values:
+
+  * Packing Policy (partial, complete)
+  * Shipping Policy (prepaid, manual, postpaid, picking)
```

Comment:
Change this to: Shipping Policy (Deliver each product when available or Deliver all products at once)